template_engine/jinja2_engine.py

```diff
@@ -12,18 +12,19 @@ def get_jinja_env(force_filesystemloader=False):
         env = jinja2.Environment(
             auto_reload=False,
             loader=jinja2.ModuleLoader(os.path.join(os.path.dirname(__file__), '../templates_jinja2_compiled.zip')),
-            extensions=['jinja2.ext.autoescape'],
+            extensions=['jinja2.ext.autoescape', 'jinja2.ext.with_'],
```

Comment:
If we update to `jinja2==2.9` we don't need to pull in these extensions anymore - but upgrading our jinja to a new version felt out of scope for this work. I'll probably file a ticket to bump that dependency.